content/learn/contribute/helping-out/reviewing-pull-requests.md

```diff
@@ -42,19 +42,19 @@ Otherwise, please try to avoid asking for additional work unless absolutely nece
 With the large number of contributors and reviewers collectively building a complex system, disagreements will inevitably occur. Regardless of whether the disagreement is about naming, performance, or code coverage, the goal should be to reach a compromise amicably.
 
 - **Keep scope in mind**. Trivial cleanup of surrounding area is okay to recommend, but don't expect a contributor to overhaul a portion of Bevy on their own.
-- **Don't be afraid to escalate**. Regardless of what side of the review process you're on, if you feel someone is being unreasonable don't hesitate to tag one of the maintainers or post in a related dev channel in the Discord.
+- **Don't be afraid to escalate**. Regardless of what side of the review process you're on, if you feel someone is being unreasonable don't hesitate to tag one of the Maintainers or post in a related dev channel in the Discord.
 - **Not all review comments will necessarily be addressed**, and that's okay. Feel free to open an issue for further discussion.
 
 ## What to review
 
 There are three main places you can check for things to review:
 
-1. Pull requests which are ready and in need of more reviews on [bevy](https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+-label%3AS-Ready-For-Final-Review+-draft%3A%3Atrue+-label%3AS-Needs-RFC+-reviewed-by%3A%40me+-author%3A%40me).
-2. Pull requests on [bevy](https://github.com/bevyengine/bevy/pulls) and the [bevy-website](https://github.com/bevyengine/bevy-website/pulls) repos.
+1. Pull requests which are ready and in need of more reviews on the [bevy repository](https://github.com/bevyengine/bevy/pulls?q=is%3Aopen+is%3Apr+-label%3AS-Ready-For-Final-Review+-draft%3A%3Atrue+-label%3AS-Needs-RFC+-reviewed-by%3A%40me+-author%3A%40me).
+2. Pull requests on the [bevy](https://github.com/bevyengine/bevy/pulls) and the [bevy-website](https://github.com/bevyengine/bevy-website/pulls) repositories.
```

Comment:
I've been noticing a lot of inconsistency between capitalized "Bevy" and lowercase "bevy". We should decide on one and standardize it throughout our documentation (in a follow-up, ofc!).

Reply:
The difference is when we're referring to Bevy, the proper noun, and bevy, the name of the specific GitHub repository in which the engine lives.

Reply:
Probable best way to deal with this is encapsulating all the repository references (e.g., `bevy`, `bevy-website`, etc.) in backticks \`\`, AKA single-line code blocks.